proposals/0002-root-signature-in-clang.md

```diff
@@ -898,6 +918,12 @@ Operands:
 
     When the Filter of a StaticSampler is `FILTER_COMPARISON*`,
     the ComparisonFunc cannot be 0.
+  
+  - Valid values for Flags
+    - For root signature version 1.0 and 1.1 must be 0
+    - For root signature version 1.2:
+      - SAMPLER_FLAG_UINT_BORDER_COLOR
```

Comment:
When SAMPLER_FLAG_UINT_BORDER_COLOR is set the border color must be on eof:

* opaque black uint
* opaque white uint
* transparent black

I think that this also suggests that we need to add these colors as well because they're not currently listed in the grammar section of this doc at least.  See https://github.com/microsoft/DirectX-Headers/blob/b8238e686c4f8acfe234e9800e89612444f7b758/include/directx/d3d12.h#L4087